INSTALL.md

```diff
@@ -8,16 +8,33 @@ You must first install development and libraries to your host. Assuming Ubuntu
 (apply similar to RHEL derivatives):
 
 ```
-$ sudo apt-get update && sudo apt-get install -y build-essential libssl-dev uuid-dev
+$ sudo apt-get update && sudo apt-get install -y build-essential libssl-dev uuid-dev libgpgme-dev
```

Comment:
why do we need libgpgme again ?

Reply:
I don't remember why, I think it was from some docker image building thing?

Anyway, the proper package to install on Ubuntu is `libgpgme11-dev`